.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Continuous Integration
+
+on:
+  pull_request:
+    branches: [main] # run on PRs opened against `main`
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        node: [16]
```

Comment:
Have we validated that it indeed runs against Node v16 as in `.nvmrc` file?

@alayek can you please add a step that prints output of `node --version`?

Reply:
You were right. I've now added an extra read step for `.nvmrc`